WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingUseCase.swift

```diff
@@ -86,6 +91,16 @@ final class CardPresentPaymentsOnboardingUseCase: CardPresentPaymentsOnboardingU
     func updateState() {
         state = checkOnboardingState()
     }
+
+    func selectPlugin(_ plugin: CardPresentPaymentsPlugin) {
```

Comment:
I found this hard to read, because of all the different plugins we're juggling. I think it would be improved by renaming `plugin` to `selectedPlugin` here.